pipelines/internal/commands/run/run.go

```diff
@@ -492,6 +499,54 @@ func listOf(input string) []string {
 	return strings.Split(input, ",")
 }
 
+func expandZones(project string, input []string) ([]string, error) {
+	var zones, prefixes []string
+	for _, zone := range input {
+		if strings.HasSuffix(zone, "*") {
+			prefixes = append(prefixes, zone[:len(zone)-1])
+		} else {
+			zones = append(zones, zone)
+		}
+	}
+	if len(prefixes) > 0 {
+		allZones, err := listZones(project)
+		if err != nil {
+			return nil, err
+		}
+		for _, zone := range allZones {
+			for _, prefix := range prefixes {
+				if strings.HasPrefix(zone, prefix) {
+					zones = append(zones, zone)
+					continue
```

Comment:
Extraneous continue?

Reply:
Woops; good catch, was intended to be a break.